spec/features/consumer/shopping/products_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'spec_helper'
+
+feature "As a consumer I want to view products", js: true do
+  include AuthenticationWorkflow
+  include WebHelper
+  include ShopWorkflow
+  include UIComponentHelper
+
+  describe "Viewing a product" do
+
+    let(:distributor) { create(:distributor_enterprise, with_payment_and_shipping: true) }
+    let(:supplier) { create(:supplier_enterprise) }
+    let(:oc1) { create(:simple_order_cycle, distributors: [distributor], coordinator: create(:distributor_enterprise), orders_close_at: 2.days.from_now) }
+    let(:product) { create(:simple_product, supplier: supplier) }
+    let(:variant) { product.variants.first }
+    let(:order) { create(:order, distributor: distributor) }
+    let(:exchange1) { oc1.exchanges.to_enterprises(distributor).outgoing.first }
+
+    before do
+      set_order order
+    end
+
+    describe "viewing HTML product descriptions" do
+
```

Comment:
Just a tiny code style thing: for consistency with the rest of the code base, could I get you to remove these empty lines at the beginning and end of the describe blocks?